libs/notification/mui/package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@croz/nrich-notification-mui",
   "description": "Contains the UI implementation of notifications for the MUI component library",
-  "version": "1.0.0",
+  "version": "1.0.2",
```

Comment:
Version here jumped by 2 so please check if this is wanted

Reply:
Updated all the packages to the initial versions (1.0.0).